src/pages/signup/SignupForm.tsx

```diff
@@ -0,0 +1,270 @@
+                  />
+                  <FormFeedback>{errors.signupFirstName}</FormFeedback>
+                </FormGroup>
+              </Col>
+              <Col md={6}>
+              <FormGroup>
+                  <Label for="signupLastName" className="form-label">
+                    First Name
+                  </Label>
+                  <Input
+                    type="text"
+                    name="signupLastName"
+                    id="signupLastName"
+                    tag={Field}
+                    onChange={(event: React.FormEvent<HTMLInputElement>) => {
+                      setFieldValue('signupLastName', event.currentTarget.value);
+
+                      if (this.state.providerError !== null) {
+                        this.setState({ providerError: null });
+                      }
+                    }}
+                    placeholder="Last Name"
+                    invalid={!!errors.signupLastName && touched.signupLastName}
+                  />
+                  <FormFeedback>{errors.signupLastName}</FormFeedback>
+                </FormGroup>
+              </Col>
+            </Row>
+
+            <FormGroup>
+              <Label for="signupEmail" className="form-label">
+                Email Address
+              </Label>
+              <Input
+                type="email"
+                name="signupEmail"
+                id="signupEmail"
+                tag={Field}
+                onChange={(event: React.FormEvent<HTMLInputElement>) => {
+                  setFieldValue('signupEmail', event.currentTarget.value);
+
+                  if (this.state.providerError !== null) {
+                    this.setState({ providerError: null });
+                  }
+                }}
+                placeholder="Email address"
+                invalid={!!errors.signupEmail && touched.signupEmail}
+              />
+              <FormFeedback>{errors.signupEmail}</FormFeedback>
+            </FormGroup>
+
+            <FormGroup>
+              <Label for="signupPassword" className="form-label">
+                Password
+              </Label>
+              <Input
+                type="password"
+                name="signupPassword"
+                id="signupPassword"
+                tag={Field}
+                onChange={(event: React.FormEvent<HTMLInputElement>) => {
+                  setFieldValue('signupPassword', event.currentTarget.value);
+
+                  if (this.state.providerError !== null) {
+                    this.setState({ providerError: null });
+                  }
+                }}
+                placeholder="********"
+                invalid={!!errors.signupPassword && touched.signupPassword}
+              />
+              <FormFeedback>{errors.signupPassword}</FormFeedback>
+            </FormGroup>
+
+            <FormGroup>
+              <Label for="signupConfirmPassword" className="form-label">
+                Confirm Password
```

Comment:
do we need a confirm password?

Reply:
i dont think it's bad to have it.  i think users expect/dont mind a confirm password field, i feel it adds a more secure feeling to the site.  we can remove if the design calls for it not being there?